lib/yform.php

```diff
@@ -75,6 +75,7 @@ public function __construct(array $params = [])
         $this->objparams['output'] = ''; // Final output of form
 
         // predefined dataset via sql
+        $this->objparams['db_connection_id'] = 1; // ID of db connection in config.yml
```

Comment:
Im Core nennen wir den Parameter in der Regel `$dbId`, entsprechend würde ich hier auch nur `db_id` nehmen, ohne `connection`.